omod/src/main/webapp/pages/visit/templates/encounters/checkInEncounterHeading.gsp

```diff
@@ -1,4 +1,19 @@
-<span class="encounter-name title encounter-span"><i ng-show="icon" class="{{ icon }}"></i> {{ encounter.encounterType.display }}</span> <!-- encounter-type class added for smoke tests -->
-<span class="date-span"><i class="icon-calendar"></i>{{ encounter.encounterDatetime | serverDate:DatetimeFormats.date }}</span>
-<span class="time-span"><i class="icon-time"></i>{{ encounter.encounterDatetime | serverDate:DatetimeFormats.time }}</span>
-<span class="encounter-provider provider-and-location-span">{{ encounter.obs | byConcept:Concepts.typeOfVisit:true | obs }}</span>
\ No newline at end of file
+<div class="row justify-content-between">
+    <div class="encounter-type col-8">
+        <i ng-show="icon" class="{{ icon }}"></i> {{ encounter.encounterType.display }}
+    </div>
+    <div class="col-4 text-right">
+        <span class="date-span"><i class="icon-calendar"></i>{{ encounter.encounterDatetime | serverDate:DatetimeFormats.date }}</span>
+        <span class="time-span"><i class="icon-time"></i>{{ encounter.encounterDatetime | serverDate:DatetimeFormats.time }}</span>
+    </div>
+</div>
+<div class="row col-12">
+    <span class="encounter-card-summary">{{ encounter.obs | byConcept:Concepts.typeOfVisit:true | obs }}</span>
+</div>
+<div class="row justify-content-between">
+    <div class="col-8">
+        <span class="encounter-card-summary provider-and-location-span">{{ 'emr.by' | translate }} {{ encounter.encounterProviders | encounterRole:primaryEncounterRoleUuid | getProviderName }}</span>
+        <span class="encounter-card-summary admission-location provider-and-location-span" ng-hide="encounter.location.uuid == encounter.visit.location.uuid">${ ui.message("uicommons.at") } {{encounter.location | omrsDisplay}}</span>
```

Comment:
Are there autoformatters for GSP? Does ctrl+alt+L in IntelliJ do anything for this file? These lines are a little bit long

Reply:
Good question.  This is copied directly from "defaultEncounterHeading".  These should really use a single template and pull them in, but I chose not to refactor, just to be consistent.  I'll clean them up.